molecule/default/tests/test_openstack_services.py

```diff
@@ -38,6 +46,42 @@ def service_enabled(service, run_on_host):
     else:
         return False
 
+
+def get_openstack_release(run_on_host):
```

Comment:
This function will be used in future tests, wondering if you could put it in other place for universal access like a fixture in conftest.py or in pytest-rpc?

Reply:
Definitely could be ported to pytest-rpc.